packages/perseus/src/util/interactive.ts

```diff
@@ -99,11 +100,7 @@ _.extend(GraphUtils.Graphie.prototype, {
         );
 
         const mouselayerZIndex = 2;
-        graph.mouselayer = Raphael(
```

Comment:
Fun fact, we create `graph.raphael` by passing in the `this.el`, so as far as I could tell it's always the same as `graph.raphael.canvas.parentNode`. This reduces how much mocking we need to do quite a bit. 

I also verified that Storybook worked for all of our graph widgets.